hax/test/integration/test_motr.py

```diff
@@ -313,18 +314,20 @@ def my_get(key: str, recurse: bool = False):
                  json.dumps({"name": "ios", "state": "failed"})),
                 ('m0conf/nodes/0x6e00000000000001:0x3/processes'
                  '/0x7200000000000001:0xa/services/0x7300000000000001:0xc',
-                 json.dumps({"name": "ios", "state": "failed"}))]
+                 json.dumps({"name": "ios", "state": "failed"})),
+                ('m0conf/nodes/0x6e00000000000001:0x3',
+                 json.dumps({"name": "localhost", "state": "failed"}))]
             ]
         elif key == 'm0conf/sites' and recurse:
             return [new_kv(k, v) for k, v in [(
                 'm0conf/sites/0x5300000000000001:0x1/racks'
                 '/0x6100000000000001:0x2/encls/0x6500000000000001:0x4'
                 '/ctrls/0x6300000000000001:0x5',
-                json.dumps({"state": "M0_NC_UNKNOWN"})),
+                json.dumps({"state": "failed"})),
```

Comment:
This `failed` value doesn't correspond to [4/KV RFC](https://github.com/Seagate/cortx-hare/blob/179616eff3c17fbda776623b2a0f2a4c64412e97/rfc/4/README.md):

> ## Consul KV Schema
> 
> Key | Value | Description
> --- | --- | ---
> `m0conf/sites/<site_fid>` | `{ "state": "<HA state>" }` | [HA state](#ha-state) of this site.
> `m0conf/sites/<site_fid>/racks/<rack_fid>` | `{ "state": "<HA state>" }` | [HA state](#ha-state) of this rack.
> `m0conf/sites/<site_fid>/racks/<rack_fid>/encls/<encl_fid>` | `{ "node": "<node_fid>", "state": "<HA state>" }` | Fid of the corresponding node and [HA state](#ha-state) of this enclosure.
> `m0conf/sites/<site_fid>/racks/<rack_fid>/encls/<encl_fid>/ctrls/<ctrl_fid>` | `{ "state": "<HA state>" }` | [HA state](#ha-state) of this controller.
> `m0conf/sites/<site_fid>/racks/<rack_fid>/encls/<encl_fid>/ctrls/<ctrl_fid>/drives/<drive_fid>` | `{ "dev": "<sdev_fid>", "state": "<HA state>" }` | Fid of the corresponding storage device and [HA state](#ha-state) of this drive.
> 
> <...>
> ### HA state
> 
> See [`enum m0_ha_obj_state`](https://github.com/Seagate/cortx-motr/blob/dev/ha/note.h#L119) in Motr code, `ha/note.h`.




Does it mean that the RFC is outdated? Or does your PR violate this RFC?

Reply:
You are correct, we are violating the RFC. 
We need to change the `set_process_state()` and `set_sdev_state()` functions too, as they also follow same convention.